operator/scripts/build-testing-docker-images.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+VERSION="${1:-latest}"
```

Comment:
@vmuzikar changed the defaults so that they can be more convenient as soon as 18 release is out.